Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM thelounge/thelounge:3.0.0
+
+COPY config.js /var/opt/thelounge/
+
+EXPOSE 9000
```

Comment:
I don't think this line is necessary

Reply:
The `EXPOSE`? That's not, but useful when testing locally? I guess in this case we can use `docker build -P ...` to map it (not sure though)?

Reply:
If you need this for local testing, it should be possible to specify it as a command-line option. It doesn't belong in the Dockerfile.